torchsde/settings.py

```diff
@@ -52,3 +52,7 @@ class LEVY_AREA_APPROXIMATIONS(metaclass=ContainerMeta):  # noqa
     space_time = 'space-time'  # Only compute an (exact) space-time Levy area
     davie = 'davie'  # Compute Davie's approximation to Levy area
     foster = 'foster'  # Compute Foster's correction to Davie's approximation to Levy area
+
+
+class METHOD_OPTIONS(metaclass=ContainerMeta):
+    grad_free = 'grad_free'
```

Comment:
There should be a new line at the end of this file. (Something somethin PEP8 something something.)

Reply:
Done!